### DISCUSSION TITLE
芝杜Z9S下的OpenWrt无法成功安装阿里云盘webdav插件，请messense帮忙

### DISCUSSION THREAD

**Discussion by oyboven** [messense/aliyundrive-webdav#318] [Q&A] [answered]
路由器型号：芝杜Z9S,安卓+openwrt双系统

opkg print-architecture 查看cup框架
root@OpenWrt:~# opkg print-architecture
arch all 1
arch noarch 1
arch rtd1295 10
而Z9S产品说明中标识
CPU：Realtek RTD1296 ARM Cortex-A53四核
GPU：ARM T820 MP3(3核)
于是选择了
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.2.5/aliyundrive-webdav_1.2.5-1_arm_cortex-a5_vfpv4.ipk
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.2.5/luci-app-aliyundrive-webdav_1.2.5_all.ipk
wget https://github.com/messense/aliyundrive-webdav/releases/download/v1.2.5/luci-i18n-aliyundrive-webdav-zh-cn_1.2.5-1_all.ipk
下载然后改下 /etc/opkg.conf 增加
arch all 1
arch noarch 1
arch rtd1295 10
arch arm_cortex-a5_vfpv4 100

执行下面命令安装
opkg install aliyundrive-webdav_1.2.5-1_arm_cortex-a5_vfpv4.ipk
opkg install luci-app-aliyundrive-webdav_1.2.5_all.ipk
opkg install luci-i18n-aliyundrive-webdav-zh-cn_1.2.5-1_all.ipk
虽然能安装成功，但进入OpenWrt的服务菜单下，点击“阿里云盘WebDav”进入界面，发现仅显示“阿里云盘WebDav”文字，没有相关配置界面和日志界面，请问是选择的ipk不正确吗还是什么原因无法显示和配置呢？特请messense帮忙解决，或别的开发者提供解决帮助，非常感谢！

**Comment by jjm2473** [comment, marked as answer]
应该要下载a53的ipk。
然后执行这个命令安装：
```
opkg --add-arch all:1 --add-arch noarch:1 --add-arch aarch64_cortex-a53:5 --add-arch rtd1295:10 install aliyundrive-webdav_*.ipk
```

其他两个ipk直接安装就行。
